### PR TITLE
Mobilenet V3 and fastscnn develop WIP

### PR DESCRIPTION
**Pull request by johnzja**
(i) This PR fixes problems of inverted residual module in fast-scnn. inverted residual module is moved to mmseg/models/utils, and is kept even with mmcls.
(ii) Some problems may still exist in mobilenet v3. The latest model is still training.
(iii) Unit tests are needed for MobileNet v3.

### CODE REVIEW

**Comment by xvjiarui** [comment]
We may separate MobileNetV3 and Fast-SCNN fix into 2 PRs.

**Comment by johnzja** [comment]
> We may separate MobileNetV3 and Fast-SCNN fix into 2 PRs.

Problems in Fast-SCNN do not affect correct training & Testing. You can merge this PR until MobileNetV3 is complete.

File: mmseg/models/utils/inverted_residual.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import torch.nn as nn
+import torch.utils.checkpoint as cp
+from mmcv.cnn import ConvModule
+
+from .se_layer import SELayer
+
+
+class InvertedResidual(nn.Module):
```
Comment (xvjiarui) [inline mmseg/models/utils/inverted_residual.py:8]: There is also one `InvertedResidual` in `mmseg/model/utils/`. 
We should have a consistent one.

Comment (johnzja) [reply mmseg/models/utils/inverted_residual.py:8]: I've merged these two "inverted residual" files. Now the final version of "InvertedResidual" is in mmseg/model/utils only.

**Pull request closed without merging**